apps/src/lib/ui/ParentLetter.jsx

```diff
@@ -0,0 +1,237 @@
+          </li>
+        </ol>
+        <h1>Step 4 - Review Code.org’s privacy policy</h1>
+        <p>
+          Code.org assigns utmost importance to student safety and security.
+          Code.org has signed the{' '}
+          <a href={PRIVACY_PLEDGE_URL}>Student Privacy Pledge</a> and their
+          privacy practices have received{' '}
+          <a href={COMMON_SENSE_ARTICLE_URL}>
+            one of the highest overall scores from Common Sense Media
+          </a>
+          . You can find further details by viewing Code.org’s{' '}
+          <a href={PRIVACY_POLICY_URL}>Privacy Policy</a>.
+        </p>
+        <p>
+          Computer science teaches students critical thinking, problem solving,
+          and digital citizenship, and benefits all students in today’s world,
+          no matter what opportunities they pursue in the future.
+        </p>
+        <p>
+          Please let me know if you have any questions and thank you for your
+          continued support of your child and of our classroom!
+        </p>
+        <p>{teacherName}</p>
+      </article>
+    </div>
+  );
+}
+ParentLetter.propTypes = {
+  loginType: PropTypes.oneOf(Object.values(SectionLoginType)).isRequired,
+  sectionCode: PropTypes.string,
+  teacherName: PropTypes.string.isRequired
+};
+
+const Header = () => {
+  return (
+    <header style={{backgroundColor: color.teal, marginBottom: 30}}>
+      <img
+        src="/shared/images/CodeLogo_White.png"
+        style={{
+          height: 30,
+          margin: 15
+        }}
+      />
+    </header>
+  );
+};
+
+const SignInInstructions = ({loginType, sectionCode}) => {
+  let steps;
+  switch (loginType) {
+    case SectionLoginType.clever:
+      steps = (
+        <ol>
+          <li>
+            Have your students log in to their Clever account at{' '}
+            <a href="https://www.clever.com/">www.clever.com</a> (click "Sign in
+            as a student" at the top right)
+          </li>
+          <li>
+            Click on the Code.org logo on the Clever dashboard. The logo looks
+            like this:
+            <br />
+            <img
+              src="/shared/images/clever_code_org_logo.png"
+              style={{
+                width: 60,
+                margin: 10
+              }}
+            />
+          </li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.google_classroom:
+      steps = (
+        <ol>
+          <GoToSignIn />
+          <li>Choose 'Sign in with Google'</li>
+          <li>Sign in via the Google sign-in dialog</li>
+        </ol>
+      );
+      break;
+
+    case SectionLoginType.picture:
+      steps = (
+        <ol>
+          <GoToSectionSignIn sectionCode={sectionCode} />
+          <li>Click on their picture password and then click 'Sign in'</li>
+          <li>
+            If your student does not remember their picture password, please
+            email me and I will provide it
+          </li>
```

Comment:
tiny nit: punctuation here? and line 181

Reply:
Which punctuation are you expecting? The convention in the spec was no punctuation at the end of any list items.